index.html

```diff
@@ -2433,21 +2455,55 @@ <h2>
           <ol class="algorithm">
             <li>Set the [=assigned launch consumer=] to |consumer|.
             </li>
-            <li>If the [=unconsumed launch params=] on the {{Window/window}} is
-            set:
+            <li>[=list/For each=] [=unconsumed launch params=]:
+            </li>
+            <li>
+              <ol>
+          <li>Call |consumer| with [=unconsumed launch params=].
```

Comment:
as dmurph told me, you can format this with `tidy -config tidyconfig.txt -m index.html`

Reply:
Oh cool thanks, done.